posthog/hogql/printer.py

```diff
@@ -1100,6 +1102,21 @@ def visit_call(self, node: ast.Call):
                 args_count = len(node.args) - func_meta.passthrough_suffix_args_count
                 node_args, passthrough_suffix_args = node.args[:args_count], node.args[args_count:]
 
+                if node.name in SURVEY_FUNCTIONS:
+                    if node.name == "getSurveyResponse":
+                        if not isinstance(node_args[0], ast.Constant):
+                            raise QueryError("getSurveyResponse first argument must be a constant")
+                        question_index = node_args[0].value
```

Comment:
```suggestion
                        question_index_obj = node_args[0]
                        if not isinstance(question_index_obj, ast.Constant):
                            raise QueryError("getSurveyResponse first argument must be a constant")
                        question_index = question_index_obj.value
```

or just get rid of `question_index` now